src/opcode.rs

```diff
@@ -175,7 +175,7 @@ opcode!(
         len: { u32 },
         ;;
         ioprio: u16 = 0,
-        offset64: libc::off64_t = 0,
+        offset64: u64 = 0,
```

Comment:
Can we change it back to `offset`?

Reply:
I should have confirmed that kernel always accepts u64 offset, i don't know why liburing uses `off_t`. maybe we need to check with @axboe .

It is good to keep it as it is for conservative reasons.

Reply:
That said, it'd probably be easier for non-c/c++ if we just made them explicitly 64 bits on any build. I'd be fine with doing that, looks like it's just impacting fallocate and madvise/fadvise.

Reply:
Thank you for @axboe advice! I think it is always good to use 64bit

Reply:
@axboe mainly my doubt was about signed/unsigned because `off_t` is signed, so with this PR we remove `off_t` and use `u64` as liburing.

The only one left is `fallocate` where liburing uses `off_t`, but then it casts it to `u64`. Should we use `u64` as well?

Reply:
Yes, I'll change fallocate to just use 64-bit offset/length unconditionally.